app/src/pages/inside/logsPage/logItemInfo/logItemInfoTabs/logItemInfoTabs.jsx

```diff
@@ -34,6 +36,7 @@ import StackTraceIcon from 'common/img/stack-trace-inline.svg';
 import AttachmentIcon from 'common/img/attachment-inline.svg';
 import InfoIcon from 'common/img/info-inline.svg';
 import ClockIcon from 'common/img/clock-inline.svg';
+import LogIcon from 'common/img/log-view-inline.svg';
```

Comment:
According to the design, the icon for the LOG uses a different one
![image](https://user-images.githubusercontent.com/34712661/115236013-84456600-a123-11eb-8d86-22b597d668a7.png)
Is its change included within the scope of this story?

Reply:
Ilya advised to use an existing icon